client/src/main/java/com/tigrisdata/db/client/TigrisCollection.java

```diff
@@ -242,6 +242,21 @@ DeleteResponse delete(TigrisFilter filter, DeleteRequestOptions deleteRequestOpt
    */
   CollectionDescription describe(CollectionOptions collectionOptions) throws TigrisException;
 
+  /**
+   * Publishes messages into the collection.
+   *
+   * @param messages list of messages to publish
+   * @return an instance of {@link PublishResponse} from server
+   * @throws TigrisException in case of an error
+   */
+  PublishResponse<T> publish(List<T> messages) throws TigrisException;
```

Comment:
I thought we will have a different class for PubSub where we will have these APIs so as to avoid confusion with other methods on document type?

Reply:
@mattayres how you modeled in ts - I think that will be better. 

client --> db --> [collection, topic]

Reply:
Yeah, we should follow that model. This was an early revision.